src/internal/typestyle.ts

```diff
@@ -31,6 +31,7 @@ export class TypeStyle {
   private _pendingRawChange: boolean;
   private _raw: string;
   private _tag?: StylesTarget;
+  private _window: Window = window;
```

Comment:
Sadly (I think) this would `TypeError` for non window environments. Need something like `typeof window === 'undefined' ? undefined : window`.